core/src/main/java/org/infinispan/remoting/inboundhandler/BasePerCacheInboundInvocationHandler.java

```diff
@@ -255,6 +272,46 @@ public long getAsyncXSiteRequestsReceived() {
       return statisticsEnabled ? asyncXsiteReceived.sum() : 0;
    }
 
+   @ManagedAttribute(description = "Returns the number of Cross-Site requests queued and running",
+         displayName = "Number of Cross-Site Requests running or queued.",
+         units = Units.NONE,
+         displayType = DisplayType.SUMMARY)
+   public long getAsyncXSitePendingRequests() {
```

Comment:
IMO the stats should be on `BaseBackupReceiver`, and the backup receiver should create the action sequencer as well, instead of fetching it from the `AsyncBackupHandler`.